### PR TITLE
[docs] Fixed broken links at Getting Started

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description
Fixed broken links at Getting Started.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix
summary: Fixed broken links at Getting Started.
impact_level: low
```

### CODE REVIEW

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.liquid [hunk 1]
```diff
@@ -120,7 +120,7 @@
    - не менее 4 ядер CPU;
    - не менее 8 ГБ RAM;
    - не менее 60 ГБ дискового пространства на быстром диске (400+ IOPS);
-   - [поддерживаемая ОС](/products/kubernetes-platform/documentation/v1/supported_versions.html);
+   - [поддерживаемая ОС](https://deckhouse.ru/products/kubernetes-platform/documentation/v1/supported_versions.html);
```
Comment (z9r5) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.liquid:123]:
```suggestion
   - [поддерживаемая ОС](/products/kubernetes-platform/documentation/v1/supported_versions.html);
```

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.liquid [hunk 2]
```diff
@@ -91,7 +91,7 @@
 <li>
 {% if page.platform_type == "existing" %}
   <p><strong>Существующий кластер Kubernetes.</strong></p>
-  <p>Версии Kubernetes и ОС узлов кластера должны быть <a href="../../products/kubernetes-platform/documentation/v1/supported_versions.html">в списке поддерживаемых</a>.</p>
+  <p>Версии Kubernetes и ОС узлов кластера должны быть <a href="https://deckhouse.ru/products/kubernetes-platform/documentation/v1/supported_versions.html">в списке поддерживаемых</a>.</p>
```
Comment (z9r5) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.liquid:94]:
```suggestion
  <p>Версии Kubernetes и ОС узлов кластера должны быть <a href="/products/kubernetes-platform/documentation/v1/supported_versions.html">в списке поддерживаемых</a>.</p>
```

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid [hunk 3]
```diff
@@ -112,7 +112,7 @@ Scheme of Deckhouse installation in a private environment:<br />
    - at least 4 CPU cores
    - at least 8 GB of RAM
    - at least 60 GB of disk space for the cluster and etcd data on a fast disk (400+ IOPS)
-   - [supported OS](/products/kubernetes-platform/documentation/v1/supported_versions.html)
+   - [supported OS](https://deckhouse.io/products/kubernetes-platform/documentation/v1/supported_versions.html)
```
Comment (z9r5) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid:115]:
```suggestion
   - [supported OS](/products/kubernetes-platform/documentation/v1/supported_versions.html)
```

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid [hunk 4]
```diff
@@ -83,7 +83,7 @@ Scheme of Deckhouse installation in a private environment:<br />
 <li>
 {% if page.platform_type == "existing" %}
   <p><strong>The existing Kubernetes cluster.</strong></p>
-  <p>The Kubernetes versions and OS of cluster nodes should be on <a href="../../products/kubernetes-platform/documentation/v1/supported_versions.html">the list of supported</a>.</p>
+  <p>The Kubernetes versions and OS of cluster nodes should be on <a href="https://deckhouse.io/products/kubernetes-platform/documentation/v1/supported_versions.html">the list of supported</a>.</p>
```
Comment (z9r5) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid:86]:
```suggestion
  <p>The Kubernetes versions and OS of cluster nodes should be on <a href="/kubernetes-platform/documentation/v1/supported_versions.html">the list of supported</a>.</p>
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid [hunk 5]
```diff
@@ -83,7 +83,7 @@ Scheme of Deckhouse installation in a private environment:<br />
 <li>
 {% if page.platform_type == "existing" %}
   <p><strong>The existing Kubernetes cluster.</strong></p>
-  <p>The Kubernetes versions and OS of cluster nodes should be on <a href="../../products/kubernetes-platform/documentation/v1/supported_versions.html">the list of supported</a>.</p>
+  <p>The Kubernetes versions and OS of cluster nodes should be on <a href="/kubernetes-platform/documentation/v1/supported_versions.html">the list of supported</a>.</p>
```
Comment (z9r5) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid:86]:
```suggestion
  <p>The Kubernetes versions and OS of cluster nodes should be on <a href="/products/kubernetes-platform/documentation/v1/supported_versions.html">the list of supported</a>.</p>
```

**Pull request merged**